src/vcpkg/base/unicode.cpp

```diff
@@ -92,6 +92,59 @@ namespace vcpkg::Unicode
         return count;
     }
 
+    const char* utf8_decode_code_point(const char* first, const char* last, char32_t& out) noexcept
+    {
```

Comment:
I double checked that this extraction does not effectively revert https://github.com/microsoft/vcpkg-tool/pull/265/

Reply:
Should this be static?

Reply:
no, it's used outside the file (hence why it's been extracted)